website/components/Navbar/index.js

```diff
@@ -0,0 +1,54 @@
+import NavSection from 'components/NavSection';
+import StyledLink from 'components/StyledLink';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { SubNav, SubNavSections } from '@strapi/design-system/SubNav';
+import { Box } from '@strapi/design-system/Box';
+import { Icon } from '@strapi/design-system/Icon';
+import Discourse from '@strapi/icons/Discourse';
+import Github from '@strapi/icons/Github';
+import Image from 'next/image';
+import styled from 'styled-components';
+import { useTheme } from '@strapi/design-system/ThemeProvider';
+
+const HorizontalLine = styled.div`
+  background: ${(props) => props.theme.colors.neutral200};
+  width: 1.5rem;
+  height: 1px;
+  margin: 0 0 0 1.5rem;
+`;
+
+const Navbar = ({ navigationContent }) => {
+  const theme = useTheme();
```

Comment:
You don't need the theme here :)

Reply:
@Bassel17 Can you remove the theme here please

Reply:
You forgot this one @Bassel17